scania-digital-design-system/tegel
# test: playwright setup

This PR adds the initial setup for using Playwright as testing framework for our components. 

It installs "playwright" and "stencil-playwright", the second one being a wrapper to allow for the use of web components with Playwright.

It also adds some basic/initial tests for the Accordion. 

**How to test:**
1. Check out the branch
2. cd into `packages/core`
3. Run `npm install`
4. Run `npx playwright test`
